test/distributed/org/apache/cassandra/distributed/test/cql3/SingleNodeTableWalkTest.java

```diff
@@ -130,7 +130,7 @@ protected List<CreateIndexDDL.Indexer> supportedIndexers()
             }
         }
         Select select = builder.build();
-        return state.command(rs, select, (wholePartition ? "Whole Partition" : "Single Row"));
+        return state.command(rs, select, (wholePartition ? "By Partition Key" : "By Primary Key"));
```

Comment:
Abe pointed out on slack that "whole partition" isn't true with "LIMIT" and "PER PARTITION LIMIT", so renamed